README.md

```diff
@@ -146,9 +146,9 @@ outside this list)
         signing.gnupg.passphrase=${password_for_imported_sumoapi_key}
         ```
 2. Remove `-SNAPSHOT` suffix from `version` in `build.gradle`
-3. Make a release branch with Scala version and project version, ex. `elasticsearch-client-7.0.0-M19`:
+3. Make a release branch with Scala version and project version, ex. `elasticsearch-client-7.1.1`:
     ```
-    export RELEASE_VERSION=elasticsearch-client-7.0.0-M19
+    export RELEASE_VERSION=elasticsearch-client-7.1.1
```

Comment:
Btw. do we have any conventions about the versions?

Reply:
I assume that there is a backward-incompatible change (spray migration) and a change of building / releasing method (gradle) that should indicate a larger version change (`7.0.X-XXX` -> `7.1.X`). But this is only my assumption. I don't think there is a strict convention for this project. I think rising it to `7.1.0` is sharp enough to make users aware of compatibility changes, and also gentle enough to not indicate the change of logic.